src/backend/main.ts

```diff
@@ -1127,13 +1127,13 @@ if (existsSync(installed)) {
   })
 }
 
-ipcMain.handle('refreshLibrary', async (e, library?) => {
+ipcMain.handle('refreshLibrary', async (e, library?: Runner | 'all') => {
```

Comment:
this shouldn't be needed. it is inferred from the ipcBridge.d.ts declaration

Reply:
ok, this was showing an error before and now its not so maybe was some typescript cache, not sure.